deno.json

```diff
@@ -19,21 +19,28 @@
   },
   "compilerOptions": { "jsx": "react-jsx", "jsxImportSource": "preact" },
   "imports": {
-    "@/": "./",
     "$fresh/": "https://raw.githubusercontent.com/denoland/fresh/60220dd33b5b0f6b5c72927c933dbc32a3c4734e/",
-    "preact": "https://esm.sh/preact@10.19.2",
-    "preact/": "https://esm.sh/preact@10.19.2/",
-    "preact-render-to-string": "https://esm.sh/*preact-render-to-string@6.2.2",
+    "@/": "./",
+    "@deno/gfm": "jsr:@deno/gfm@^0.9",
     "@preact/signals": "https://esm.sh/*@preact/signals@1.2.1",
     "@preact/signals-core": "https://esm.sh/*@preact/signals-core@1.5.0",
-    "tailwindcss": "npm:tailwindcss@3.4.1",
-    "tailwindcss/": "npm:/tailwindcss@3.4.1/",
-    "tailwindcss/plugin": "npm:/tailwindcss@3.4.1/plugin.js",
-    "$std/": "https://deno.land/std@0.208.0/",
-    "stripe": "npm:/stripe@13.5.0",
+    "@std/assert": "jsr:@std/assert@^1.0",
+    "@std/datetime": "jsr:@std/datetime@^0.225",
+    "@std/front-matter": "jsr:@std/front-matter@^1.0",
+    "@std/fs": "jsr:@std/fs@^1.0",
+    "@std/http": "jsr:@std/http@^1.0",
+    "@std/path": "jsr:@std/path@^1.0",
+    "@std/testing": "jsr:@std/testing@^1.0",
+    "@std/ulid": "jsr:@std/ulid@^1.0",
+    "fresh_charts/": "https://deno.land/x/fresh_charts@0.3.1/",
     "kv_oauth/": "https://deno.land/x/deno_kv_oauth@v0.9.1/",
+    "preact": "https://esm.sh/preact@10.19.2",
+    "preact/": "https://esm.sh/preact@10.19.2/",
+    "stripe": "npm:/stripe@13.5.0",
     "tabler_icons_tsx/": "https://deno.land/x/tabler_icons_tsx@0.0.4/tsx/",
```

Comment:
also a note on this, we can switch this to:

> "@tabler/icons-preact": "npm:@tabler/icons-preact@^3.14.0",

The module we are using is deprecated: https://github.com/hashrock/tabler-icons-tsx?tab=readme-ov-file#deprecated

Reply:
happy to do this in this PR or a seperate one